GiniVision.podspec

```diff
@@ -18,6 +18,11 @@ The Gini Vision Library for iOS provides functionality to capture documents with
   s.social_media_url = 'https://twitter.com/gini'
   s.swift_version    = '5.0'
   s.ios.deployment_target = '10.0'
+  
+  # workaround for https://github.com/CocoaPods/CocoaPods/issues/9521
+  # Remove after iOS 11 is dropped.
+  s.xcconfig = { 'ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES' => 'YES' }
```

Comment:
What effect will this have on client app's ipa size? 🤔

Reply:
This is what normally should happen. This line just forces it. You can see the setting in Build Settings. I'm still looking for ways to fix it without this line, but it works without side effects as far as I can tell.